src/object/srv_ec.c

```diff
@@ -322,3 +329,73 @@ ec_copy_iods(daos_iod_t *in_iod, int nr, daos_iod_t **out_iod)
 out:
 	return rc;
 }
+
+
+#ifdef UNIT_TEST
+struct daos_oclass_attr oca =
+	{
+		.ca_schema		= DAOS_OS_SINGLE,
+		.ca_resil		= DAOS_RES_EC,
+		.ca_grp_nr		= 1,
+		.u.ec			= {
+			.e_k		= 2,
+			.e_p		= 2,
+			.e_len		= 1 << 15,
+		},
+	};
+
+int main(int argc, char* argv[])
+{
+	struct ec_bulk_spec* skip_list[1];
+        daos_handle_t oh = {0L};
+        daos_handle_t th = {1L};
+        daos_key_t dkey;
+        dkey.iov_buf = "42";
```

Comment:
(style)  code indent should use tabs where possible
(style)  Missing a blank line after declarations